endpoints/users.py

```diff
@@ -264,3 +265,13 @@ def apply_for_diary(id):
     db.session.commit()
 
     return json.dumps(diary[0].serialize(), cls=ModelEncoder), 201
+
+
+@app.route("/users/<id>/pointlog", methods=['GET'])
+def get_user_point_log(id):
+    data = []
+    rows = ClanPointsLog.query.filter_by(user_id=id).all()
```

Comment:
```suggestion
    rows = ClanPointsLog.query.filter_by(user_id=id).order_by(ClanPointsLog.timestamp.desc()).all()
```

Can order by timestamp in the filter so we don't have to sort afterwards